src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,31 @@
-export const PostInfo = () => (
-  <>Put the post here</>
-);
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => {
+  const {
+    title,
+    user,
+    body,
+    comments,
+  } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+
+        {user && (
+          <UserInfo user={user} />
+        )}
+      </div>
+
+      <p className="PostInfo__body">{body}</p>
+
+      {comments.length > 0
```

Comment:
Pls [see falsy values doc](https://developer.mozilla.org/en-US/docs/Glossary/Falsy)
```suggestion
      {comments.length 
```